infra/aws/terraform/iam/k8s-infra-prow-canary/terraform.tfvars

```diff
@@ -1,23 +0,0 @@
-/*
-Copyright 2023 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-region = "us-east-2"
-
-eks_infra_admins = [
```

Comment:
FYI - having access provisioned this way could allow someone to supply a different list of admins than what is shown here. I suspect these changes are manually applied by only a select group of users so less of a concern, but its still something to be aware of.